src/Elcodi/Store/GeoBundle/Form/LocationSelectorBuilder.php

```diff
@@ -95,6 +97,14 @@ protected function buildRootSelector($selectedOption = null)
         $rootLocations   = $this
             ->locationProvider
             ->getRootLocations();
+
+        if (empty($rootLocations)) {
+            throw new HttpException(
```

Comment:
I don't like to throw boundary specific exceptions in inner layers. I'd prefer a custom exception (`EmptyRootLocationsException`) or a generic one instead (`\RuntimeException`).